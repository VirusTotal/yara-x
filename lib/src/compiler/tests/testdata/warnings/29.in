import "test_proto2"

rule test {
	condition: 1 == test_proto2.array_bool[0]
}