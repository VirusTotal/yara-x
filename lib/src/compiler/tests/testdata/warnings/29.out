warning[bool_int_comparison]: comparison between boolean and integer
 --> line:4:13
  |
4 |  condition: 1 == test_proto2.array_bool[0]
  |             ------------------------------ this comparison can be replaced with: `test_proto2.array_bool[0]`
  |
